rosdep/python.yaml

```
@@ -4334,6 +4334,48 @@ python-rpi.gpio-pip:
   ubuntu:
     pip:
       packages: [RPi.GPIO]
+python3-rpi.gpio:
+  debian:
+    buster: [python3-rpi.gpio]
```

Galactic is targeting Bullseye:
```suggestion
    bullseye: [python3-rpi.gpio]
    buster: [python3-rpi.gpio]
```